### PR TITLE
Update tutorial to use new DDT data and download FITS files in the RTD build

### LINKED ISSUE #348: Update tutorial to use new DDT data
We should update the tutorial to use the data from pid_2_114/115 which is some unembargoed DDT from 2023.

Here are some good datasets I have found:

##  2023/10/16 18:45 - 2023/10/16 18:48 
<details>
<summary> Fido search </summary>

```python
from sunpy.net import Fido, attrs as a
import dkist.net

Fido.search(a.dkist.Proposal("pid_2_114"), a.Time("2023/10/16 18:45", "2023/10/16 18:48"))
```

</details>

* [AJQWW](https://dkist.data.nso.edu/datasetview/AJQWW) - VBI - 3 timesteps of 9 frame mosaics, seeing is *excellent*.
* ([BEEWM](https://dkist.data.nso.edu/datasetview/BEEWM)[1] [AGJZM](https://dkist.data.nso.edu/datasetview/AGJZM) [AKPWX](https://dkist.data.nso.edu/datasetview/AKPWX)) - VBI - co-observed with above in different passbands.
* ([AEEXW](https://dkist.data.nso.edu/datasetview/AEEXW), [ADDPJ](https://dkist.data.nso.edu/datasetview/ADDPJ), [BZYJD](https://dkist.data.nso.edu/datasetview/BZYJD)) - VISP - Stokes I only, 4 map repeats of 2611 raster steps, cobserved with AJQWW etc. - These are O(25Gb) unfortunately.

## 2023/10/16 18:21 - 2023/10/16 18:41
<details>
<summary> Fido search </summary>

```python
from sunpy.net import Fido, attrs as a
import dkist.net

Fido.search(a.dkist.Proposal("pid_2_114"), a.Time("2023/10/16 18:21", "2023/10/16 18:41"))
```

</details>

* ([BXYGP](https://dkist.data.nso.edu/datasetview/BXYGP), [BWVEV](https://dkist.data.nso.edu/datasetview/BWVEV)[1], [ADDYP](https://dkist.data.nso.edu/datasetview/ADDYP))  - VBI - Not mosaicked
* ([BLDMN](https://dkist.data.nso.edu/datasetview/BLDMN), [BJQZW](https://dkist.data.nso.edu/datasetview/BJQZW), [BKPLX](https://dkist.data.nso.edu/datasetview/BKPLX)) - VISP - Full Stokes, good seeing, small dataset (one 425 step raster).

[1] Images aren't very clear on this dataset as that passband has major reconstruction issues.

## Other's I found on my travels

* [BDDXM](https://dkist.data.nso.edu/datasetview/BDDXM) - VBI - 2 timestep of mosaic, seeing is excellent

### PR DESCRIPTION
**Pull request by Cadair**
Fixes #348

### CODE REVIEW

File: docs/howto_guides/reproject_vbi_moasic.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+```{code-cell} ipython3
+first_tiles = [tile[0] for tile in ds.flat]
+```
+
+```{code-cell} ipython3
+:tags: [skip-execution]
+
+for i, tile in enumerate(first_tiles):
+    tile.files.download(wait=i == len(first_tiles) - 1)
```
Comment (SolarDrew) [inline docs/howto_guides/reproject_vbi_moasic.md:73]: Maybe add a comment here explaining what the wait argument is doing in this case, unless this is demonstrated elsewhere in the docs. Took me a minute to parse this.

File: docs/howto_guides/reproject_vbi_moasic.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    ax = fig.add_subplot(ds.shape[0], ds.shape[1], i+1, projection=tile.wcs)
+    ax.set_title(f"MINDEX1={tile.headers[0]['MINDEX1']}, MINDEX2={tile.headers[0]['MINDEX2']}")
+    ax.imshow(tile.data)
+
+fig.tight_layout()
+```
+
+## Regridding with Reproject
+
+First, we need the most recent version of reproject (released on Friday) for this to work, so let's check:
```
Comment (SolarDrew) [inline docs/howto_guides/reproject_vbi_moasic.md:91]: This clarification is going to date very quickly :P

File: docs/howto_guides/reproject_vbi_moasic.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+Make a new `NDCube` object and plot it.
+
+```{code-cell} ipython3
+plt.figure(figsize=(10,10))
+stitched = NDCube(arr, reference_wcs)
+stitched.plot()
+```
+
+We can see some small artifacting which can be hidden by using the `match_background=True` keyword argument to `reproject_and_coadd`.
```
Comment (SolarDrew) [inline docs/howto_guides/reproject_vbi_moasic.md:148]: Is it worth explaining more what this argument does?

File: docs/tutorial/3_the_dataset.md [hunk 4]
```diff
@@ -152,33 +150,24 @@ A useful feature of the `Dataset` class, which it inherits from `NDCube` is the
...
 
 Notice again that this has reduced the dimensionality of the world coordinates as well as of the data itself.
-
-Finally, we can plot our cropped dataset to show the feature we were looking at.
-
-```{code-cell} python
-import matplotlib.pyplot as plt
-
-cropped.plot()
-plt.show()
```
Comment (SolarDrew) [inline docs/tutorial/3_the_dataset.md:183]: We're not showing the data here any more?

**Review by SolarDrew — CHANGES_REQUESTED** [review]

File: docs/tutorial/3_the_dataset.md (hunk 4 above)
Comment (Cadair) [reply docs/tutorial/3_the_dataset.md:183]: Well we haven't downloaded it yet, and it's slow so I thought we could drop it?

File: docs/howto_guides/reproject_vbi_moasic.md (hunk 3 above)
Comment (Cadair) [reply docs/howto_guides/reproject_vbi_moasic.md:148]: Perhaps we should just remove this line, you don't really see the artifacting in this data.

**Comment by AlisdairDavey** [comment]
The link https://dkistdc-public-documentation--349.com.readthedocs.build/projects/python-tools/en/349/howto_guides/reproject_vbi_moasic.html has "mosaic" spelt incorrectly. In addition the menu on the left hand side of the page also had "mosaic" spelt incorrectly.

**Review by SolarDrew — APPROVED** [review]
LGTM :+1:

**Pull request merged**